src/scala/main/ly/stealth/mesos/kafka/scheduler/http/api/BrokerApi.scala

```diff
@@ -68,7 +68,7 @@ trait BrokerApiComponentImpl extends BrokerApiComponent {
           Response.ok(BrokerStatusResponse(brokerNodes))
             .build()
         case Failure(e) =>
-          Response.status(Response.Status.BAD_REQUEST).build()
+          Response.status(Response.Status.BAD_REQUEST).entity(HttpErrorResponse(400, e.getMessage())).build()
```

Comment:
Can you just refactor Status.BadRequest(...) to do this instead of copy/pasting it to each place it's used?